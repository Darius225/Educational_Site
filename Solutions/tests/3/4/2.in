10
31814 18323 20299 13493 8820 1075 3894 2523 400 29559 