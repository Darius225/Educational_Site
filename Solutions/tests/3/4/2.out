400 1075 2523 3894 8820 13493 18323 20299 29559 31814 